pgsqltoolsservice/capabilities/connection_options/mysql_connection_options.py

```diff
@@ -52,6 +52,16 @@
         is_required=True,
         group_name='Security'
     ),
+    ConnectionOption(
+        name='azureAccountToken',
+        display_name='Access Token',
+        description='Indicates an Active Directory access token to be used when connecting to the data source',
```

Comment:
Was this already there? or did you add it? Can you verify if AAD authentication works for MySQL?

Reply:
This has been removed.